grunt/tasks/translate/saveCourseData.js

```diff
@@ -15,7 +15,7 @@ module.exports = function (grunt) {
       "blocks",
       "components"
     ].forEach(function (filename) {
-      var src = path.join(srcPath, "course", targetLang, filename+".json");
+      var src = path.join(srcPath, "course", targetLang, filename+"."+grunt.config("jsonext"));
```

Comment:
Should we be caching `jsonext` outside the `forEach` loop like [here](https://github.com/adaptlearning/adapt_framework/pull/1684/files#diff-3bac7cf018801ee7c4228dce79ba0de7R12)?